projectroles/forms.py

```diff
@@ -444,6 +446,18 @@ def _set_app_setting_notes(self, s_field, s_val):
                 self.fields[
                     s_field
                 ].help_text += ' [Not editable on target sites]'
+        if plugin:
+            self.fields[s_field].label = format_html(
+                '{} <i class="iconify" title="{}" data-toggle="tooltip" data-icon="{}"></i>',
```

Comment:
Since the Bootstrap tooltip isn't working here, feel free to remove the `data-toggle` attributes from here and the user form.